ui/pages/confirmations/confirm-send-ether/__snapshots__/confirm-send-ether.test.js.snap

```diff
@@ -337,82 +337,6 @@ exports[`ConfirmSendEther should render correct information for for confirm send
             </div>
           </div>
         </div>
-        <div
```

Comment:
Does this suggest we're missing something in the test since we're not rendering the estimated changes anymore?

Reply:
We are not rendering the simulations loader if `simulationData` is not filled, that's the affect of the change. 

But in reality, this is already happening that way, `simulationData` is never `undefined` in the UI.